common/config/rush/version-policies.json

```diff
@@ -1,8 +1,6 @@
 [
   {
-    "policyName": "prerelease-monorepo-lockStep",
-    "definitionName": "lockStepVersion",
-    "version": "1.0.0",
-    "nextBump": "prerelease"
+    "policyName": "individual-versioning",
+    "definitionName": "individualVersion"
```

Comment:
mind reverting these changes? This is going to break the rush publish.

Reply:
do i change the version to 1.0.1 or remove the next bump is a pre-release?

Reply:
just updated them. It should be good to go now.